mkdocs.yml

```diff
@@ -0,0 +1,81 @@
+site_name: csvcubed
```

Comment:
Should this file be here? We already have `external-docs/mkdocs.yml`.

Reply:
Don't know where this came from - have deleted it